packages/worker/src/subscription.ts

```diff
@@ -0,0 +1,310 @@
+import {
+    database,
+    config,
+    interfaces,
+    subgraph,
+    utils,
+} from '@impactmarket/core';
+import { execute } from 'apollo-link';
+import { WebSocketLink } from 'apollo-link-ws';
+import { ethers } from 'ethers';
+import gql from 'graphql-tag';
+import { Op } from 'sequelize';
+import { SubscriptionClient } from 'subscriptions-transport-ws';
+import ws from 'ws';
+
+const getWsClient = function (wsurl) {
+    const client = new SubscriptionClient(wsurl, { reconnect: true }, ws);
+    return client;
+};
+
+const createSubscriptionObservable = (wsurl, query, variables) => {
+    const link = new WebSocketLink(getWsClient(wsurl));
+    return execute(link, { query, variables });
+};
+
+export const communitySubscription = async () => {
+    try {
+        const COMMUNITY_QUERY = gql`
+            subscription {
+                communityEntities(
+                    orderBy: startDayId
+                    orderDirection: desc
+                    where: {
+                      startDayId_gte: ${
+                          (new Date().getTime() / 1000 / 86400) | 0
+                      }
+                    }
```

Comment:
won't this make users get notified twice if the server is restarted?

Reply:
as there is no DateTime field, only the day, just below there is a check to see if there is already a notification created for the community, just to avoid a situation like this of the server restarting and users being notified of the communities opened today